### PR TITLE
Add Tailwind CSS

### PR DESCRIPTION
**Pull request by sauloperez**
This adds the required configuration to use TailwindCSS to the site, which I strongly believe will make it easier for us to maintain a consistent style and speed up the dev process.

Although the config has been tested loading Tailwind stylesheets in a new `<script>`, it's not been commited so we can ship iterative changes without breaking production.

More changes will follow in separate PRs that completely refactor the stylesheets.

At the same time, this is the first step towards implementing the new design suggested in https://github.com/coopdevs/coopdevs.github.io/pull/54#issuecomment-513860651

### CODE REVIEW

File: _includes/tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  theme: {
+    extend: {}
+  },
+  variants: {},
+  plugins: []
+}
```
Comment (andresgutgon) [inline _includes/tailwind.config.js]: This file is not necessary afaik until you need it. So you can 🔥

Comment (andresgutgon) [reply _includes/tailwind.config.js]: Uhm, also why is duplicated?

File: tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  theme: {
+    extend: {}
+  },
+  variants: {},
+  plugins: []
+}
```
Comment (andresgutgon) [inline tailwind.config.js]: ditto about not being necessary

image

Comment (sauloperez) [reply tailwind.config.js]: good point. I'll add when I need it.

**Comment by sauloperez** [comment]
done @andresgutgon

**Review by andresgutgon — APPROVED** [review]

**Pull request merged**